lib/improver/tests/blending/spatial_weights/test_SpatiallyVaryingWeightsFromMask.py

```diff
@@ -653,18 +653,12 @@ def test_none_masked(self, warning_list=None):
         self.cube_to_collapse.data = np.ones(self.cube_to_collapse.data.shape)
         self.cube_to_collapse.data = np.ma.masked_equal(
             self.cube_to_collapse.data, 0)
-        expected_data = np.array([[[[0.2, 0.2, 0.2],
-                                    [0.2, 0.2, 0.2]],
-                                   [[0.2, 0.2, 0.2],
-                                    [0.2, 0.2, 0.2]]],
-                                  [[[0.5, 0.5, 0.5],
-                                    [0.5, 0.5, 0.5]],
-                                   [[0.5, 0.5, 0.5],
-                                    [0.5, 0.5, 0.5]]],
-                                  [[[0.3, 0.3, 0.3],
-                                    [0.3, 0.3, 0.3]],
-                                   [[0.3, 0.3, 0.3],
-                                    [0.3, 0.3, 0.3]]]],
+        expected_data = np.array([[[0.2, 0.2, 0.2],
```

Comment:
All these unit tests have been updated to remove any extra dimensions (like threshold) that the weights don't vary across.

Reply:
Otherwise unchanged. Seem fine.